src/routes/recipe.js

```diff
@@ -0,0 +1,160 @@
+import { Router } from 'express';
+import { body, param, validationResult } from 'express-validator';
+
+import { authenticate } from '../modules/auth';
+import models from '../modules/database';
+import loggers from '../modules/logging';
+
+const router = Router();
+const log = loggers('recipe');
+const {
+  Diluent,
+  Flavor,
+  Recipe,
+  UserProfile,
+  RecipesFlavors,
+  RecipesDiluents
+} = models;
+
+/**
+ * GET Recipe
+ * @param id int
+ */
+router.get(
+  '/:id',
+  authenticate(),
+  [
+    param('id')
+      .isNumeric()
+      .isInt({ min: 1 })
+      .toInt()
+  ],
+  async (req, res) => {
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
+    }
+
+    log.info(`request for ${req.params.id}`);
+    try {
+      const result = await Recipe.findOne({
+        where: {
+          id: req.params.id
+        },
+        include: [
+          {
+            model: UserProfile,
+            required: true
+          },
+          {
+            model: RecipesFlavors,
```

Comment:
TODO: experiment with adding `as: 'flavors'` here and `as: 'diluents'` further down.  [See docs here.](http://docs.sequelizejs.com/manual/associations.html)